src/video/VideoPanel.js

```diff
@@ -0,0 +1,44 @@
+import { Panel } from '../_common';
+import React from 'react';
+import { connect } from 'react-redux';
+import YouTube from 'react-youtube';
+import VideoList from './VideoList';
+import VideoData from '../_data/VideoData';
+import * as VideoAction from '../_actions/VideoActions';
+
+@connect(state => ({video: state.video}))
+export default class VideoPanel extends React.Component {
+
+    componentDidMount() {
+        VideoData.getAll().then((response) => {
+            return response.json();
+        }).then((jsonObj) => {
+            const newList = jsonObj.items.map((v) => {
+                return {
+                    imgSrc: v.snippet.thumbnails.high.url,
+                    title: v.snippet.title,
+                    videoUrl: 'https://www.youtube.com/watch?v=' + v.snippet.resourceId.videoId
+                };
+            });
+            this.props.dispatch(VideoAction.updateVideoList(newList));
+        })
+    }
+
+    render() {
+        const url = this.props.video.get('url');
+        const title = this.props.video.get('title');
+        const videos = this.props.video.get('videos');
+
+        return (
+            <div>
+                <Panel title={title}>
+                    <YouTube url={url}/>
+                </Panel>
+                <Panel title={"Video List"}>
+                    <VideoList videos={videos} />
+                </Panel>
```

Comment:
Not sure we will do it with two Panels per component, but since I'll be reworking it anyways, we'll leave it like that.